midi-to-part-mp3s

```diff
@@ -20,17 +20,25 @@ class Part:
         return self.midi_filepath.replace(".midi", ".mp3")
 
 def track_names(midi_data):
-  return list(map(lambda track: track.name, midi_dat.tracks))
+  return list(map(lambda track: track.name, midi_data.tracks))
```

Comment:
Forgot to mention this change: I think here the name of the variable had a typo.

Reply:
yes that's totally a mistake! thanks for catching it